templates/experimental/src/lib/usdc.ts

```
@@ -0,0 +1,101 @@
+      const tx = await this.usdcContract.transfer(
+        adminAgent.agentAddress,
+        amountInWei,
+      );
+      const receipt = await tx.wait();
+      if (receipt.status !== 1) {
+        throw new Error("Transaction failed or was reverted");
+      }
+      console.warn(`Transferred ${amount} USDC to ${to}.`);
+      return `Transferred ${amount} USDC to ${to}.`;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)